apps/widget-configurator/src/app/configurator/index.tsx

```diff
@@ -47,6 +48,48 @@ const DEFAULT_STATE = {
   buyAmount: 0,
 }
 
+const COW_LISTENERS: CowEventListeners = [
+  {
+    event: CowEvents.ON_TOAST_MESSAGE,
+    handler: (event) => {
+      console.log('[configurator] 🍞 Toast message', event.message, event.data)
+
+      // You can provide a simplistic way to handle toast messages (use the "message" to show it in your app)
+      if (event.messageType === ToastMessageType.SWAP_ETH_FLOW_SENT_TX) {
+        console.error('[configurator] 🍞 Toast message: New eth flow order', event.data.tx)
```

Comment:
can you use `.info` instead of `.error` in the examples?

Reply:
bbb1bcbaae15355060e13ce2032a893ffca2f242